### PR TITLE
Update localization.json

### PR DESCRIPTION
**Pull request by Pumpken**
Updated some german lines

### CODE REVIEW

File: EDEngineer/Resources/Data/localization.json [hunk 1]
```diff
@@ -3424,15 +3424,15 @@
     "Group ingredients?": {
       "es": null,
       "pt": null,
-      "de": null,
+      "de": "Gruppenzutaten?"
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json]: requires a comma at the end of the line

Comment (Pumpken) [reply EDEngineer/Resources/Data/localization.json]: Ah jeez, first time editing something on Git. How do I change this now without pulling a new request?

Comment (msarilar) [reply EDEngineer/Resources/Data/localization.json]: you can edit this file directly

**Comment by msarilar** [comment]
great job thanks a lot

**Pull request merged**